src/structures/MessageEmbed.js

```diff
@@ -78,6 +79,7 @@ class MessageEmbed {
     }
 
     /**
+     * Represents thumbnail of a MessageEmbed
```

Comment:
```suggestion
     * Represents the thumbnail of a MessageEmbed
```
This should be consistent with the other definitions below.